batcher/aligned-sdk/src/communication/batch.rs

```diff
@@ -0,0 +1,137 @@
+use std::{collections::HashSet, time::Duration};
+
+use futures_util::StreamExt;
+use log::{debug, error};
+use tokio::time::timeout;
+
+use crate::{
+    core::errors,
+    core::types::{
+        AlignedVerificationData, BatchInclusionData, VerificationCommitmentBatch,
+        VerificationDataCommitment,
+    },
+    eth::BatchVerifiedEventStream,
+};
+
+const AWAIT_BATCH_VERIFICATION_TIMEOUT: u64 = 60;
+
+pub async fn handle_batch_inclusion_data<'s>(
+    batch_inclusion_data: BatchInclusionData,
+    aligned_verification_data: &mut Vec<AlignedVerificationData>,
+    verification_data_commitments_rev: &mut Vec<VerificationDataCommitment>,
+    stream: &mut BatchVerifiedEventStream<'s>,
```

Comment:
lets rename this parameter to something like `event_stream`
```suggestion
    event_stream: &mut BatchVerifiedEventStream<'s>,
```

Reply:
Solved in [885721a](https://github.com/yetanotherco/aligned_layer/pull/583/commits/885721a87d1d9c5e20af5afd74873dfb63a4c1a8)